client/src/client-only-routes/ShowSettings.js

```diff
@@ -57,7 +57,6 @@ const mapStateToProps = createSelector(
 
 const mapDispatchToProps = {
   createFlashMessage,
-  navigate,
```

Comment:
This is still in the propTypes... unless, you left it to be taken care of by one of the TS refactors

Reply:
Thanks, Shaun, that's fixed now.